tests/Integration/FontTest.php

```diff
@@ -379,4 +379,17 @@ public function testXmlContent()
 
         $this->assertEquals('Example PDF', $text);
     }
+
+    /**
+     * Create a header containing a Encoding that doesn't have a BaseEncoding
+     * so I test if the Font won't raise a exception because Encoding don't have BaseEncoding
```

Comment:
```suggestion
     * Create an instance of Header containing an instance of Encoding that doesn't have a BaseEncoding.
     * Test if the Font won't raise a exception because Encoding don't have BaseEncoding.
```